middleware/timings.js

```diff
@@ -13,43 +13,24 @@ module.exports = responseTime(function(req, res, time) {
         return;
     }
 
-    if (req.method === 'GET') {
-        cloudwatch
-            .putMetricData({
-                MetricData: [
-                    {
-                        MetricName: 'RESPONSE_TIME_GET',
-                        Dimensions: [
-                            {
-                                Name: 'RESPONSE_TIME',
-                                Value: 'TIME_IN_MS'
-                            }
-                        ],
-                        Unit: 'Milliseconds',
-                        Value: time
-                    }
-                ],
-                Namespace: 'SITE/TRAFFIC'
-            })
-            .send();
-    } else if (req.method === 'POST') {
-        cloudwatch
-            .putMetricData({
-                MetricData: [
+    const method = req.method.toUpperCase();
+    const isLegacy = res.getHeader('X-BLF-Legacy') === 'true';
```

Comment:
This callback gets triggered `onHeadersSent` so we can reliably check for the `X-BLF-Legacy` header even though this middleware is mounted earlier.